docs/tutorials/aws/RAG_with_DeepSeek_R1_model_on_Sagemaker.md

```diff
@@ -0,0 +1,443 @@
+# Topic
+
+
+This tutorial introduces how to build RAG in Amazon managed OpenSearch with [DeepSeek R1 model](https://huggingface.co/deepseek-ai/DeepSeek-R1).
+If you are not using Amazon OpenSearch, you can refer to [deepseek_connector_chat_blueprint](https://github.com/opensearch-project/ml-commons/blob/main/docs/remote_inference_blueprints/deepseek_connector_chat_blueprint.md).
+
+Note: You should replace the placeholders with prefix `your_` with your own value
+
+
+# Steps
+
+## 0. Preparation
+
+### Deploy DeepSeek R1 model to Sagemaker
+
+Follow this [blog](https://community.aws/content/2sG84dNUCFzA9z4HdfqTI0tcvKP/deploying-deepseek-r1-on-amazon-sagemaker) to deploy DeepSeek R1 model to Sagemaker.
```

Comment:
This is interesting. Have you followed the steps and deployed successfully in SageMaker?

Reply:
yes, the result is from test cluster, not fake one